src/main/scala/com/microsoft/hyperspace/index/IndexConstants.scala

```diff
@@ -47,4 +47,6 @@ object IndexConstants {
     val PLAIN_TEXT = "plaintext"
     val HTML = "html"
   }
+
+  val DATA_FILE_NAME_COLUMN = "_data_file_name"
```

Comment:
`private[index] val DATA_FILE_NAME_COLUMN`?

Reply:
We can not make this constant private and package specific as its definition here is in `IndexConstants.scala` which is in index package while `CreateActionBase.scala` is the place the constant is used and it is in actions package. I

Reply:
then `[hyperspace]`. The intention is to make it internal to the project.

Reply:
Makes sense - I added it.